### PR TITLE
Integrate ex_aws and proof of concept

### LINKED ISSUE #6: Introduce `ex_aws`
We could use [Ex_AWS](https://hexdocs.pm/ex_aws/ExAws.html), just following the `getting started` section.

Just the configuration should be enough to allow us to work on #4 and #5.

### PR DESCRIPTION
**Pull request by uesteibar**
Closes #6 

This PR is just a proof of concept on how to interact with your `S3 Bucket`, using [ex_aws](https://github.com/cargosense/ex_aws).

To test this proof of concept:

`cp .env.sample .env` and set the correct keys and the bucket name.

Launch `iex`
```shell
iex -S mix
```

Inside `iex`, run:
```elixir
Abba.Aws.S3.Bucket.Server.list_objects()
```

And you'll get a beautiful list of urls for all the objects in your bucket.

**Note:** I didn't write any tests since we can't even run them until we have at least one migration.

### CODE REVIEW

File: lib/abba/aws/s3/bucket/server.ex [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  end
+
+  def handle_call(:list_objects, _from, %{bucket_name: bucket_name} = state) do
+    urls = fetch_objects(bucket_name)
+            |> extract_contents(bucket_name)
+
+    {:reply, urls, state}
+  end
+
+  def list_objects() do
```
Comment (GustavoCaso) [inline lib/abba/aws/s3/bucket/server.ex]: I like to pace all the `public` functions on top before the `handle_...` but is just a suggestion

Comment (uesteibar) [reply lib/abba/aws/s3/bucket/server.ex]: sounds good to me, I'll change it!

**Comment by GustavoCaso** [comment]
So far for looks great

**Comment by uesteibar** [comment]
@GustavoCaso good to merge then?

**Comment by GustavoCaso** [comment]
@uesteibar go for it

**Pull request merged**